index.html

```diff
@@ -4,13 +4,12 @@
         <meta charset="UTF-8">
         <title>FRIEND MAKER 5000</title>
         <script src='https://aframe.io/releases/0.3.0/aframe.min.js'></script>
-        <script src='https://cdn.rawgit.com/AltspaceVR/aframe-altspace-component/v1.3.1/dist/aframe-altspace-component.js'></script>
```

Comment:
Not sure if this JS file is required -- I had to remove this because it's conflicting with the new version of the AltspaceSDK JS.